### DISCUSSION TITLE
Old Version of BinaryNinja

### DISCUSSION THREAD

**Discussion by valour01** [Vector35/binaryninja-api#2165] [Q&A] [answered]
Hi 

My current binaryninja version is 2.2.2487. I was just wondering whether it is possible to install an old version of binaryninja. Many Thanks

**Comment by jeffli678** [comment, marked as answer]
Yes, and you can do that by navigating to "Edit"->"Preferences"->"Update Channel". From there, you can select a stable/dev version to switch to. Also, if you wish to switch to a really old version that is not listed by default, go to settings and enable "Show All Versions", that will allow all history versions to be listed.

**Reply by psifertex** [reply]
See also: https://docs.binary.ninja/getting-started.html#updates
and: https://github.com/Vector35/binaryninja-api/blob/dev/python/examples/version_switcher.py (though that works best with a commercial or headless license)
